source/RunScribeDataField.mc

```diff
@@ -94,14 +96,15 @@ class RunScribeDataField extends Ui.DataField {
             mMetricCount = 4; 
         } else if (mMetric3Type != 0) {
             mMetricCount = 3;
-        } else {
+            mMetric4Type = mMetric3Type;

```

Comment:
Please, remove this in the next commit/pull-request.

Reply:
Copy paste mistake